my_build_dir/my_install_dir/sound/MuseScore_General_License.md

```diff
@@ -0,0 +1,92 @@
+# MuseScore_General.sf2
```

Comment:
Please explain why you added this license?

Reply:
Oh sorry, I accidently added it. It crept in somehow. I will remove it

Reply:
Same for my_build_dir/my_install_dir/sound/MuseScore_General.sf3